### PR TITLE
THUB-21 | @jdwjdwjdw | Person node and teaser adjustments

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- [THUB-21](https://stanfordits.atlassian.net/browse/THUB-21): People card (or basic card variant for people)
- Person node and teaser adjustments
- Admin fixup for unpublished page alert

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Urgency
- How urgent is this? (Normal, High)

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Create some people. Add a short title with the long content you want to display on the teaser.
4. Verify the cards look like the design.  We discussed and know about the short title and the long title being kind of confusing. 
5. Add a global message. Verify it works on the admin view.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- THUB-21](https://stanfordits.atlassian.net/browse/THUB-21)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

[THUB-21]: https://stanfordits.atlassian.net/browse/THUB-21?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/scss/components/person/_person-teasers.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@charset "UTF-8";
+
+.ptype-stanford-entity,
+.paragraph--type--stanford-entity {
+
+  .ds-entity--stanford-person {
```
Comment (rebeccahongsf) [inline src/scss/components/person/_person-teasers.scss:6]: @jenbreese should the Person Teaser span full width for single column/responsive? 🤔 

For the screenshot, my screen width was 640px
image

File: src/scss/components/person/_person-teasers.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            @include thub-type-d;
+
+            background: $thub-gradient-lagunita-to-ochre;
+            -webkit-background-clip: text;
+            -webkit-text-fill-color: transparent;
+
+            &:hover,
+            &:focus,
+            &:active {
+              background:  $thub-color-ochre-dark;
```
Comment (rebeccahongsf) [inline src/scss/components/person/_person-teasers.scss:38]: Underline is missing on hover/focus 👀 

image

**Review by rebeccahongsf — CHANGES_REQUESTED** [review]
Just these fixes and we should be good to go @jenbreese! 😃

File: src/scss/components/person/_person-teasers.scss (hunk 1 above)
Comment (jenbreese) [reply src/scss/components/person/_person-teasers.scss:6]: @rebeccahongsf No, Kerri wanted the person teaser only to have the max width of 400. I forgot to put in the jira ticket to the PR. I'm sorry about that. https://stanfordits.atlassian.net/browse/THUB-21

File: src/scss/components/person/_person-teasers.scss (hunk 2 above)
Comment (jenbreese) [reply src/scss/components/person/_person-teasers.scss:38]: @rebeccahongsf I see an underline on hover on my person teaser. Can you pull and refresh? 
Screenshot 2024-07-09 at 10 34 03 AM

**Comment by jenbreese** [comment]
@rebeccahongsf Can you try this last one? I think it is good to go.  Last PR for THUB :)

**Review by rebeccahongsf — APPROVED** [review]
LGTM 🚢

**Pull request merged**